### PR TITLE
Chen formula

### LINKED ISSUE #7: Chen formula
http://www.thepokerbank.com/strategy/basic/starting-hand-selection/chen-formula/

https://poker.stackexchange.com/questions/2687/how-to-use-or-not-use-chen-formula-valuation-in-different-scenarios

### PR DESCRIPTION
**Pull request by azhikai**
Closes #7 

The formula is only used for scoring different **starting** hands only. It should not be used as a complete substitute for proper **pre-flop** starting hand strategy, but could be used as one of the heuristics. However, this is till a rather decent formula to work out **pre-flop** starting hand strengths. 

Implementation is based on http://www.thepokerbank.com/strategy/basic/starting-hand-selection/chen-formula/

### CODE REVIEW

File: chen_formula.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
+from math import ceil
+from pypokerengine.engine.card import Card
+
+# Do not use `Card.RANK_MAP.get(12)` as it would return the Card's character, e.g. 1,2,...10,J,Q,K,A
+ACE = 14
+KING = 13
+QUEEN = 12
+JACK = 12
```
Comment (zulq9) [inline chen_formula.py]: Typo on value of JACK. 
JACK = 11

Comment (azhikai) [reply chen_formula.py]: Thanks for spotting it, will fix. Will also add additional test case for it.

**Review by darren96 — DISMISSED** [review]
💯

**Review by zulq9 — DISMISSED** [review]
🔥 💯

**Review by darren96 — APPROVED** [review]

**Review by zulq9 — APPROVED** [review]
👍

**Pull request merged**